pkg/agent/openflow/client.go

```diff
@@ -294,6 +294,26 @@ type Client interface {
 	UninstallTrafficControlReturnPortFlow(returnOFPort uint32) error
 
 	InstallMulticastGroup(ofGroupID binding.GroupIDType, localReceivers []uint32) error
+
+	// InstallMulticlusterNodeFlows installs flows to handle cross-cluster packets between a regular
+	// Node and a local Gateway.
+	InstallMulticlusterNodeFlows(
+		gatewayID string,
+		peerConfigs map[*net.IPNet]net.IP,
+		tunnelPeerIP net.IP) error
+
+	// InstallMulticlusterGatewayFlows installs flows to handle cross-cluster packets between Gateways.
+	InstallMulticlusterGatewayFlows(
+		gatewayID string,
+		peerConfigs map[*net.IPNet]net.IP,
+		tunnelPeerIP net.IP,
+		localGatewayIP net.IP) error
+
+	// InstallMulticlusterClassifierFlows installs flows to classify cross-cluster packets.
+	InstallMulticlusterClassifierFlows(gatewayID string, tunnelOFPort uint32, isGateway bool) error
+
+	// UninstallMulticlusterFlows removes flows to classify cross-cluster packets.
+	UninstallMulticlusterFlows(cacheKey string) error
```

Comment:
I don't understand why only this flow uses a quite different argument. It's hard to associate this method with others.

Reply:
This is stale one which I forgot to update, rename it to `gatewayID` as others now.

Reply:
didn't find update